src/prepare_buildhost.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+test "$(lsb_release -si)" == "Ubuntu" || { echo "This script supports only Ubuntu"; exit 1; }
+set -e
+
+sudo apt-get update
+sudo apt-get install -y curl software-properties-common
+
+REL=$(lsb_release -sr | cut -d'.' -f1)
+JRUBY_VERSION=$(cat .jruby-version || echo "1.7.25")
+
+if [ $REL -lt 16 ]; then
+    sudo apt-add-repository -y ppa:openjdk-r/ppa
+    sudo apt-get update
+fi
+
+sudo apt-get install -y openjdk-8-jdk-headless build-essential git unzip debhelper
+sudo update-ca-certificates -f
+
+cd ~
+
+if ! command -v rvm &>/dev/null; then
+    gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+    curl -L https://get.rvm.io | bash -s stable
+    source ~/.rvm/scripts/rvm
+fi
+
+rvm install jruby-$JRUBY_VERSION --binary
+rvm jruby-$JRUBY_VERSION do jgem install jruby-jars:$JRUBY_VERSION bundler warbler:1.4.9 bundler-audit
+
+mkdir -p /var/tmp/xroad
+
+if [[ $REL -ge 16 && ! -e /.dockerenv ]]; then
```

Comment:
Yes, on Ubuntu 16 it will build both deb and rpm packages in Docker. On Ubuntu 14 the build behaves as it was and only deb pakcages are built natively.